### PR TITLE
fix(contributing): switch testing to pytest

### PR DESCRIPTION
**Pull request by alexcg1**
Based on discussion in Slack group, adjusted testing section

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ Locally you can do unittest via:
 
 ```bash
 pip install ".[match-py-ver]"
```
Comment (JoanFM) [inline CONTRIBUTING.md:231]: We need to change to `pip install .[test]`

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Comment by bwanglzu** [comment]
runned `pip install [.test]` and then `pytest -s -v ...`, got:
image

**Comment by JoanFM** [comment]
> runned `pip install [.test]` and then `pytest -s -v ...`, got:
> image

Then, we need to rewrite these tests so that they do not depend on these `Min and Max Rankers`, we need to change them by some `MockRankers` done for the tests.

**Comment by JoanFM** [comment]
> > runned `pip install [.test]` and then `pytest -s -v ...`, got:
> > image
> 
> Then, we need to rewrite these tests so that they do not depend on these `Min and Max Rankers`, we need to change them by some `MockRankers` done for the tests.

I will do that

**Comment by JoanFM** [comment]
Created PR #1208

**Comment by alexcg1** [comment]
This sounds like a bigger job than first envisaged. Ping me when we're good to move forward! :)

**Comment by JoanFM** [comment]
the `pip install .[test]` shpuld be put instead of the `[py_match_ver]`, that for sure

**Comment by JoanFM** [comment]
You should be able to proceed @alexcg1 with the changes requested

File: CONTRIBUTING.md (hunk 1 above)
Comment (hanxiao) [inline CONTRIBUTING.md:231]:
```suggestion
pip install ".[test]"
```

**Pull request merged**